authentication/views.py

```diff
@@ -9,15 +9,15 @@
 from django.contrib.auth import login, logout, authenticate
 from django.contrib.auth.forms import UserCreationForm, AuthenticationForm
 from django.contrib.auth.mixins import LoginRequiredMixin
- 
+
 from .forms import UserSignUp
 
-class DashboardView(LoginRequiredMixin, TemplateView):
-    login_url = '/login/'
-    redirect_field_name = 'redirect_to'
 
+class DashboardView(LoginRequiredMixin, TemplateView):
```

Comment:
there are a ton of different login_required decorators/mixins. I personally prefer using the login_required decorator at the url.py level so at a glance you can see which ones are login required and not, and organize them by function (ie open to the public, vs auth required). This isn't a suggest to change your code, but rather just telling you what I would do.

Reply:
I was racking my brain trying to figure out what the preferred way to do this was. Ill look at some of your other apps and see how you implement the decorators 👍 Thanks for the heads up.